.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,3 +21,5 @@
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 - [ ] My code follows the code style of this project,
 - [ ] My change requires a change to the documentation.
+- [ ] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md)
```

Comment:
WDYT about merging this into the first point?

```
- [ ] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project
```

Reply:
Hey @warpech. I've made the change. Please re-review.